build.js

```diff
@@ -258,93 +266,118 @@ function getNumberOfParentFolders(path, root = "html") {
 	}
 	return counter;
 }
-/**
+
+/** Precompiles JSON parsed interview objects into a list elements used in navigation
  * 
- * @param {Array} obj 
+ * @param {Array} objects JSON parsed interview objects
+ * @returns Compiled HTML code for modal navigation
  */
 function transpileInterviewNavigation(objects) {
-	str_builder = ``;
-	objects.map(obj => {
-		str_builder += `<li id="${obj.Name.replace(" ", "_")}_menu" class="nav-elem"><div>${obj.Name}</div></li>\n`;		
-	});
-	return str_builder;
+	try {
+		str_builder = ``;
+		objects.forEach(obj => {
+			str_builder += `<li id="${obj.Name.replace(" ", "_")}_menu" class="nav-elem"><div>${obj.Name}</div></li>\n`;		
+		});
+		return str_builder;
+	} catch (err) {
```

Comment:
what's the point of this catch? If an errors is thrown, it should probably stop processing anything and throw an error that stops the entire transpilation, right?

Reply:
It will stop processing, I use the catch so I can write out what the error is. This way it's easier for me to identify the problem/issue and resolve it faster.